### PR TITLE
Additional e2e tests

### PR DESCRIPTION
**Pull request by Tal-or**
The following E2E were added:
- Verify that ```NumaResourceTopology``` CRD is deployed.
- Check that nothing happens when ```NUMAResourcesOperator``` CR deployed with the wrong name.
- Check that ```NUMAResourceOperator``` CR is deleted, redeploy with different parameters and without polluting cluster state.

These tests are part of the ```[Install]``` labels tests but should run on a separate lane dedicated for them.

### CODE REVIEW

**Comment by Tal-or** [comment]
/retest
looks like infra issue

**Comment by Tal-or** [comment]
/retest
infra issue

**Comment by Tal-or** [comment]
/hold
some improvements are needed

File: test/e2e/install/bla.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package install
```
Comment (fromanirh) [inline test/e2e/install/bla.go]: `bla.go`?

Comment (Tal-or) [reply test/e2e/install/bla.go]: LOL, will fix it in a minute

Comment (Tal-or) [reply test/e2e/install/bla.go]: OK the file has been deleted in a later commit, but just to be extra righteous I'll clean the history

**Comment by Tal-or** [comment]
/hold cancel
@fromanirh please LGTM'd again if you're happy

**Pull request merged**